internal/translation/encryptionatrest/conversion_test.go

```diff
@@ -0,0 +1,50 @@
+package encryptionatrest
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	fuzz "github.com/google/gofuzz"
+	"github.com/stretchr/testify/require"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1/common"
+)
+
+func TestRoundtrip_EncryptionAtRest(t *testing.T) {
```

Comment:
Sample for making this < go native Fuzz test while still using `gofuzz`:

https://github.com/mongodb/mongodb-atlas-kubernetes/pull/1884/files#diff-845d71c11ed89ac707e9ff64ff4273b11ab5f62125de863d4c0af3317e8f5086R23